core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -583,7 +583,7 @@ object AclCommand extends Logging {
       if (options.has(commandConfigOpt) && !options.has(bootstrapServerOpt))
         CommandLineUtils.printUsageAndDie(parser, "The --command-config option can only be used with --bootstrap-server option")
 
-      if (options.has(authorizerPropertiesOpt) && options.has(bootstrapServerOpt))
+      if (options.has(authorizerPropertiesOpt) && options.has(authorizerOpt))
```

Comment:
authorizerPropertiesOpt will only need for authorizerOpt. Right?

Reply:
Yes, correct.